### DISCUSSION TITLE
Map language in Serbian - wrong/ incomplete

### DISCUSSION THREAD

**Discussion by ZeljkoBG** [organicmaps/organicmaps#7695] [Ideas]
The language of the map for the Serbian language is **Local names**.

The names on the map of Serbia should be in two letters **Serbian Cyrillic** and **Serbian Latin**.

OSM tag for Serbian Cyrillic - **name:sr**
OSM tag for Serbian Latin - **name:sr-Latn**

Please add both letters to the application

**Comment by biodranik** [comment]
name:sr should already be displayed/supported on the map and can be enabled in the app translations by adding them into data/strings/ files.

Adding sr-Latn can be complicated for a number of reasons. One is that there is a hard limit of 64 languages in OM. Increasing it will require refactoring the map data format. Another is that these two different spellings actually should be interpreted as a single language in the search and on the map, and that is tricky.

**Comment by ZeljkoBG** [comment]
I'm not sure I understand well, but there are map language selections: Serbian Cyrillic and Serbian Latin in other applications. For example, the latest version of the OsmAnd app.
